server/events/yaml/raw/project.go

```diff
@@ -106,10 +114,26 @@ func validProjectName(name string) bool {
 
 func validApplyReq(value interface{}) error {
 	reqs := value.([]string)
-	for _, r := range reqs {
-		if r != ApprovedApplyRequirement && r != MergeableApplyRequirement && r != UnDivergedApplyRequirement {
-			return fmt.Errorf("%q is not a valid apply_requirement, only %q, %q and %q are supported", r, ApprovedApplyRequirement, MergeableApplyRequirement, UnDivergedApplyRequirement)
+	for _, req := range reqs {
+		isValid := false
+		for _, applyRequirement := range applyRequirements {
+			if req == applyRequirement {
+				isValid = true
```

Comment:
why not just use a map instead of a list, this would become one line of code then?

Reply:
Hmm that's true! I'll change it to a map.

Reply:
Looks like using a map makes the `buildValidApplyRequirementsString()` unpredictable causing the tests to fail occasionally. Should I just revert back to using a slice or should I change how we're generating the test statement?

Reply:
You can probably just account for that failure by sorting the result.

Reply:
Done!